game/game.go

```diff
@@ -698,7 +713,9 @@ func (g *Game) playTurn(t int) error {
 		// We back up the board and bag since there's a possibility
 		// this play will have to be taken back, if it's a challenged phony.
 		g.board.SaveCopy()
-		g.board.PlayMove(m, g.gaddag, g.bag.LetterDistribution())
+		ld := g.bag.LetterDistribution()
+		g.board.PlayMove(m, ld)
+		g.cross_set_gen.UpdateCrossSetsForMove(m)
```

Comment:
Just for consistency sake, and I believe it’s idiomatic in Go, could you change this to `crossSetGen`?